bop_toolkit_lib/tests/unit_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Please change the name of this file to reflect what it is testing (e.g. "eval_bop19_pose_test.sh").